uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-balance-tab-filters.component.spec.js

```diff
@@ -62,69 +67,83 @@ define([
               period.current = false;
             });
 
-            expected = ctrl.absencePeriods.reduce(function (a, b) {
-              return moment(a.end_date).isAfter(b.end_date) ? a : b;
+            period = ctrl.absencePeriods.reduce(function (periodA, periodB) {
+              return moment(periodA.end_date).isAfter(periodB.end_date)
+                ? periodA
+                : periodB;
             });
 
             controllerOnChanges('absencePeriods', absencePeriods);
           });
 
           it('selects the most recent absence period', function () {
-            expect(ctrl.filters.absencePeriod).toBe(expected);
+            expect(ctrl.filters.absence_period).toBe(period.id);
           });
         });
       });
 
       describe('when absence types are updated', function () {
+        var type;
+
         beforeEach(function () {
-          expected = absenceTypes.reduce(function (a, b) {
+          type = absenceTypes.reduce(function (a, b) {
             return a.title.localeCompare(b.title) ? a : b;
           });
 
           controllerOnChanges('absenceTypes', absenceTypes);
         });
 
         it('selects the first absence type sorted by title', function () {
-          expect(ctrl.filters.absenceType).toBe(expected);
+          expect(ctrl.filters.absence_type).toBe(type.id);
+        });
+      });
+
+      describe('when logged in contact id is updated', function () {
```

Comment:
this looks odd, it reads as a `describe` and `it` title together. What does it even mean "when logged in contact id is updated"?